src/Geolocator.Plugin.WindowsPhone81/GeolocatorImplementation.cs

```diff
@@ -172,6 +172,26 @@ public Task<Position> GetPositionAsync(TimeSpan? timeout, CancellationToken? can
         }
 
         /// <summary>
+        /// Gets position async and reverse geocode
+        /// </summary>
+        /// <returns>Address of the current position</returns>
+        public Task<Address> ReverseGeocodeCurrentLocation()
+        {
+            throw new NotImplementedException();
```

Comment:
Maybe this: https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Maps.UWP/GeocoderBackend.cs ?